62.CAD/vertex_shader.hlsl

```diff
@@ -418,10 +382,103 @@ PSInput main(uint vertexID : SV_VertexID)
+                isInDrawSection = !(patternIdx & 0x1);
+            }
+
+            if (isInDrawSection)
+            {
+                outV.setColor(lineStyle.color);
+                const float sdfLineThickness = screenSpaceLineWidth / 2.0f;
+                outV.setLineThickness(sdfLineThickness);
+                
+                const double2 circleCenter = vk::RawBufferLoad<double2>(drawObj.geometryAddress, 8u);
+                const float2 v = vk::RawBufferLoad<float2>(drawObj.geometryAddress + sizeof(double2), 8u);
+                const float cosHalfAngleBetweenNormals = vk::RawBufferLoad<float>(drawObj.geometryAddress + sizeof(double2) + sizeof(float2), 8u);
+
+                const float2 circleCenterScreenSpace = transformPointScreenSpace(clipProjectionData.projectionToNDC, circleCenter);
+                outV.setPolylineConnectorCircleCenter(circleCenterScreenSpace);
+
+                // find intersection point vertex
+                float2 intersectionPoint = v * antiAliasedLineWidth;
+                intersectionPoint += circleCenterScreenSpace;
+
+                // Find other miter vertices
+                const float sinHalfAngleBetweenNormals = sqrt(1.0f - (cosHalfAngleBetweenNormals * cosHalfAngleBetweenNormals));
+                const float32_t2x2 rotationMatrix = float32_t2x2(cosHalfAngleBetweenNormals, -sinHalfAngleBetweenNormals, sinHalfAngleBetweenNormals, cosHalfAngleBetweenNormals);
+                const float2 V1 = normalize(mul(v, rotationMatrix)) * antiAliasedLineWidth;
+                const float2 V2 = normalize(mul(rotationMatrix, v)) * antiAliasedLineWidth;
+                float2 screenSpaceV1 = circleCenterScreenSpace + V1;
+                float2 screenSpaceV2 = circleCenterScreenSpace + V2;
+
+                // Pass the precomputed trapezoid values for the sdf
+                {
+                    float vLen = length(v);
+                    float2 intersectionDirection = v / vLen;
+
+                    float longBase = sinHalfAngleBetweenNormals;
+                    float shortBase = max((vLen - globals.miterLimit) * cosHalfAngleBetweenNormals / sinHalfAngleBetweenNormals, 0.0);
+                    // height of the trapezoid / triangle
+                    float hLen = min(globals.miterLimit, vLen);
+
+                    outV.setPolylineConnectorTrapezoidStart(-1.0 * intersectionDirection * sdfLineThickness);
+                    outV.setPolylineConnectorTrapezoidEnd(intersectionDirection * hLen * sdfLineThickness);
+                    outV.setPolylineConnectorTrapezoidLongBase(sinHalfAngleBetweenNormals * ((1.0 + vLen) / (vLen - cosHalfAngleBetweenNormals)) * sdfLineThickness);
+                    outV.setPolylineConnectorTrapezoidShortBase(shortBase * sdfLineThickness);
+                }
+
+                if (vertexIdx == 0u)
+                {
+                    outV.position = float4(screenSpaceV1, 0.0f, 1.0f);   
```

Comment:
Compute V1 and screenSpaceV1 inside this branch. cause it's only being used here